dev-qt/qtdropbox/qtdropbox-20151109.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="An API for the well known cloud storage service"
+DESCRIPTION="Basically QtDropbox aims to provide an easy to use possibility \
+to access the REST API of Dropbox. All HTTP calls are hidden behind the \
+curtains of neat C++/Qt classes with nice method names and specific uses."
+HOMEPAGE="https://github.com/lycis/QtDropbox/"
```

Comment:
```diff
-HOMEPAGE="https://github.com/lycis/QtDropbox/"
+HOMEPAGE="https://github.com/lycis/QtDropbox"
```
If we did get a redirect (to the address that includes a trailing slash) when we request the address without the trailing slash, then yes, it would have been nice to keep the trailing slash.
But here, we don't get redirected, so this means the trailing slash is useless.
(We can therefore drop it.)